src/components/Popover/PopoverExample.tsx

```diff
@@ -1,4 +1,3 @@
-import { relative } from 'path/posix';
```

Comment:
I don't know why `tsc` flagged this on my PR rather than on the PR that introduced it, but whatev